source/docs/software/hardware-apis/pneumatics/pneumatics.rst

```diff
@@ -6,89 +6,101 @@ Operating pneumatic cylinders
 
-PCM Modules are identified by their Node ID. The default Node ID for PCMs is 0. If using a single PCM on the bus it is recommended to leave it at the default Node ID.
 
 Generating and Storing Pressure
 -------------------------------
 
-In FRC\ |reg|, pressure is created using a pneumatic compressor and stored in pneumatic tanks. The compressor doesn't necessarily have to be on the robot, but must be powered by the robot's PCM(s). The "Closed Loop" mode on the Compressor is enabled by default, and it is *not* recommended that teams change this setting. When closed loop control is enabled the PCM will automatically turn the compressor on when the pressure switch is closed (below the pressure threshold) and turn it off when the pressure switch is open (~120PSI). When closed loop control is disabled the compressor will not be turned on. Using a Compressor, users can query the status of the compressor. The state (currently on or off), pressure switch state, and compressor current can all be queried from the Compressor object.
+Pressure is created using a pneumatic compressor and stored in pneumatic tanks. The compressor doesn't necessarily have to be on the robot, but must be powered by the robot's pneumatics module. The "Closed Loop" mode on the Compressor is enabled by default, and it is *not* recommended that teams change this setting. When closed loop control is enabled the pneumatic module will automatically turn the compressor on when the digital pressure switch is closed (below the pressure threshold) and turn it off when the pressure switch is open (~120PSI). When closed loop control is disabled the compressor will not be turned on. Using the ``Compressor`` (`Java <https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/Compressor.html>`__ / `C++ <https://first.wpi.edu/wpilib/allwpilib/docs/release/cpp/classfrc_1_1_compressor.html>`__) class, users can query the status of the compressor. The state (currently on or off), pressure switch state, and compressor current can all be queried from the Compressor object.
 
-.. note:: The Pneumatics Control Module from Cross the Road Electronics allows for integrated closed loop control of a compressor. Creating any instance of a Solenoid or Double Solenoid object will enable the Compressor control on the corresponding PCM. The Compressor object is only needed if you want the ability to turn off the compressor or query compressor status.
+.. note:: The Compressor object is only needed if you want the ability to turn off the compressor, change the pressure sensor (PCH only), or query compressor status.
 
 .. tabs::
 
     .. code-tab:: java
 
-        Compressor c = new Compressor(0, PneumaticsModuleType.CTREPCM);
+        Compressor pcm = new Compressor(0, PneumaticsModuleType.CTREPCM);
+        Compressor pch = new Compressor(1, PneumaticsMOduleType.REVPH);
```

Comment:
```suggestion
        // PCM
        Compressor c = new Compressor(0, PneumaticsModuleType.CTREPCM);
        // PCH
        Compressor c = new Compressor(1, PneumaticsModuleType.REVPH);
```

For all the objects created in these examples (compressor, solenoid, double solenoid), I think it would be better to call the variable the same thing as they are both used the same way.

Reply:
I'm conflicted on this. You wouldn't write code that way, but at the same point these aren't complete examples that show a concept, more like correct syntax API snippets

Reply:
I would suggest keeping it as is. I think the context that their both Compressor objects is gotten well across.